internal/plugin/api.go

```diff
@@ -21,6 +21,6 @@ import "github.com/NVIDIA/k8s-device-plugin/internal/rm"
 // Interface defines the API for the plugin package
 type Interface interface {
 	Devices() rm.Devices
-	Start() error
+	Start(string) error
```

Comment:
why not add `kubeletSocket` as a field to the struct that implements this Interface? IMO it's better to avoid modifying the interface method signature if we can help it

Reply:
Updated.

Reply:
Thanks